src/java/voldemort/server/protocol/admin/AsyncOperation.java

```diff
@@ -29,13 +29,17 @@ public void markComplete() {
 
     public void run() {
         updateStatus("Started " + getStatus());
+        String previousThreadName = Thread.currentThread().getName();
+        Thread.currentThread().setName(previousThreadName + "; AsyncOperation ID " + status.getId() + ": " + status.getDescription());
```

Comment:
It contains a brief description of the kind of ```AsyncOperation```, for example "Fetch store".

For example, here's what a thread dump looks like, with one unused thread and one used by an AsyncOp:

```
"voldemort-scheduler-service1-t2" #28 daemon prio=5 os_prio=0 tid=0x00007f201859e800 nid=0x23c9 waiting on condition [0x00007f1fd1676000]
   java.lang.Thread.State: WAITING (parking)
...
"voldemort-scheduler-service1-t1; AsyncOperation ID 2: Fetch store" #27 daemon prio=5 os_prio=0 tid=0x00007f2018498000 nid=0x23c8 runnable [0x00007f
   java.lang.Thread.State: RUNNABLE
```